pkg/converter/constant.go

```diff
@@ -20,6 +20,9 @@ const (
 	LayerAnnotationNydusBootstrap     = "containerd.io/snapshot/nydus-bootstrap"
 	LayerAnnotationNydusSourceChainID = "containerd.io/snapshot/nydus-source-chainid"
 	LayerAnnotationNydusEncryptedBlob = "containerd.io/snapshot/nydus-encrypted-blob"
+	LayerAnnotationSourceDigest       = "containerd.io/snapshot/nydus-source-digest"
+	LayerAnnotationDestDigest 		  = "containerd.io/snapshot/nydus-dest-digest"
```

Comment:
Does this code has been formatted?

Reply:
It seems weird, because it is formatted locally. May need to push again.
![image](https://github.com/containerd/nydus-snapshotter/assets/54702708/6b23efa5-f875-4b53-886c-d98e4550e94f)

Reply:
fixed after re-push